citre-lang-c.el

```diff
@@ -158,6 +166,16 @@
          `(filter (or ,(citre-core-filter-kind "function")
                       ,(citre-core-filter-kind "macro"))
                   +)))
+       ('struct
```

Comment:
Can we put an example in the comment here? Also, I don't understand the example you gave in the OP:

```
struct point point;
```

Do people really do this?

I don't know C well, so I'm gonna trust you. Please, write a clear example, or do nothing if the logic is obvious to C programmers.

Reply:
> Do people really do this?

Yes.

```
[yamato@dev64]~/.citre% readtags -t upstream-linux.tags -Q '(and (eq? $kind "variable") (eq? $typeref (concat "struct:" $name))))' -l | wc -l
391
[yamato@dev64]~/.citre% readtags -t upstream-linux.tags -Q '(and (eq? $kind "member") (eq? $typeref (concat "struct:" $name))))' -l | wc -l 
2439
```

`struct point point` is so annoying and it appears so frequently. If I can remember correctly, you can write the same things with  "struct point" in go lang.